go.mod

```diff
@@ -5,7 +5,7 @@ go 1.20
 require (
 	github.com/OneOfOne/xxhash v1.2.8
 	github.com/edgexfoundry/go-mod-bootstrap/v3 v3.0.0-dev.47
```

Comment:
please also help bump go-mod-bootstrap to bootstrap/v3 from 3.0.0-dev.53 (not 54)